lib/ui/pedals/rings.lua

```diff
@@ -0,0 +1,91 @@
+--- RingsPedal
+-- @classmod RingsPedal
+
+local ControlSpec = require "controlspec"
+local UI = require "ui"
+local Pedal = include("lib/ui/pedals/pedal")
+local Controlspecs = include("lib/ui/util/controlspecs")
+
+local RingsPedal = Pedal:new()
+-- Must match this pedal's .sc file's *id
+RingsPedal.id = "rings"
+
+function RingsPedal:new(bypass_by_default)
+  local i = Pedal:new(bypass_by_default)
+  setmetatable(i, self)
+  self.__index = self
+
+  i.sections = {
+    {"Pitch/Struct", "Bright/Damp"}, 
+    {"Position", "Model/EasterEgg"},
+    i:_default_section(),
+  }
+  i:_complete_initialization()
+
+  return i
+end
+
+function RingsPedal:name(short)
+  return short and "RNGS" or "Rings"
```

Comment:
worried about confusion with "Ring Mod" 🤔 i'll think on it for a bit

Reply:
miRings?